lib/backend/k8s/k8s.go

```diff
@@ -648,7 +516,8 @@ func (c *KubeClient) listPolicies(ctx context.Context, l model.ResourceListOptio
 	}
 
 	// List all Global Network Policies.
-	gnps, err := c.gnpClient.List(ctx, l, revision)
+	gnpClient, _ := c.getResourceClientFromList(l)
```

Comment:
We should be handling this error

Reply:
this code is changing - so I just wanted to get it compiling.

Reply:
actually I've now removed the error - so this can return a nil interface.  I actually think in this case it's assertable that it returns something (since we know what we're putting in) so I'm not convinced we need to check that since we'll just end up panicking (which is fine IMHO).